src/pages/Resources/Resources.js

```diff
@@ -4,15 +4,23 @@ import { useQuery } from 'react-query';
 import Search from '../../components/Search';
 import { Sidebar, Main } from '../pageSections';
 import { Grid, Typography } from '@material-ui/core';
+import { Pagination } from '@material-ui/lab';
 import { ResourceCard } from './ResourceCard';
 import { getResources } from '../../utils/queries';
 
 function Resources() {
   const [searchValue, setSearchValue] = useState('');
-  const { isLoading, data, error } = useQuery(
-    [`?search=${searchValue}`],
-    getResources
-  );
+  const [goToPage, setGoToPage] = useState();
+  const [isPagination, triggerPagination] = useState();
```

Comment:
Was tempted to ask that this be renamed `paginate, setPaginate` for consistency's sake, but `triggerPagination` has a certain ring to it 🤔

Reply:
Also, thinking we should initialize the value to be null: `useState();`